aiida_vasp/calcs/vasp.py

```diff
@@ -69,24 +69,17 @@ def define(cls, spec):
         super(VaspCalculation, cls).define(spec)
         # Define the inputs.
         # options is passed automatically.
-        spec.input('parameters', valid_type=get_data_class('dict'), help='The VASP input parameters (INCAR).', required=False)
+        spec.input('parameters', valid_type=get_data_class('dict'), help='The VASP input parameters (INCAR).')
```

Comment:
Do we bump major here as this would be interface changes, at least strictly. Happy to do it. I guess with the changes of the immigrant, we anyway need to bump it. Also, we would probably recommend the use of the `ImmigrantWorkChain`, which would yield an implicit interface change.

Reply:
Also, it is possible to run `VASP` without any `parameters`, e.g. an empty `INCAR`. But it needs to be created/touched. That was why this was set to `False`. We however can require it, check content and then dump an empty file. Original thought was to be explicit to the user. Is there a good reason why this needs to be required?

I think we need to verify that the `INCAR` is created if `parameters` are not supplied, but that is a separate fix.

Reply:
I'm sorry. The spec.input change here is the reverting to the original ones from those introduced by my mistake made in the last PR. The following is those in the current develop:

https://github.com/aiida-vasp/aiida-vasp/blob/a4ed2ead6a0e7d0f51b31e2f856246c3275da6c4/aiida_vasp/calcs/vasp.py#L81-L97

Reply:
Thanks for mentioning that. I lost track of that and did not check before commented :) Sorry for the noise. Anyway, you triggered thoughts and I think a proper discussion about what should be required or not have never been made so maybe we can discuss this in an upcoming meeting and take notes in the wiki. Agreed?

Reply:
Yes. I agree very much. Thanks for your thought.